ccm_web/client/src/pages/AccessDenied.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Link, makeStyles, Typography } from '@material-ui/core'
+
+import InlineErrorAlert from '../components/InlineErrorAlert'
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    margin: theme.spacing(2)
+  }
+}))
+
+interface AccessDeniedProps {
+  email: string
+  helpLink: string
+}
+
+export default function AccessDenied (props: AccessDeniedProps): JSX.Element {
+  const classes = useStyles()
+  const emailLink = <Link href={`mailto:${props.email}`}>{props.email}</Link>
+  const forHelpLink = (
+    <Link href={props.helpLink} target='_blank' rel='noopener'>
+      ITS Help Page
+    </Link>
+  )
```

Comment:
What is the reason for having the email and link passed as props? I feel at some level you might be thinking to get them as configuration?

Reply:
You had one of them as a prop I think, so I followed that pattern. At some point we may want to make it configurable, so I guess that prepares us a little for it.